build.sh

```diff
@@ -72,19 +72,25 @@ if [ ! -f "${BUILD_ENV}" ]; then
 fi
 source ${BUILD_ENV}
 
-# Clear out old builds
-if [ -d "${GEOPM_INSTALL}" ]; then
+# Check that GEOPM_INSTALL is defined unless we are skipping install
+if [ -z "${GEOPM_INSTALL}" ] && [ -z "${GEOPM_SKIP_INSTALL+x}" ];
+    echo "Please define GEOPM_INSTALL prior to using this script" 1>&2
+    exit 1
+fi
+
+# Clear out old builds unless we are skipping install
+if [ -d "${GEOPM_INSTALL}" ] && [ -z "${GEOPM_SKIP_INSTALL+x}" ]; then
```

Comment:
This seems risky.  I have absolutely debugged issues before where a stale build was being picked up because it was in the LD_LIBRARY_PATH o and/or PATH, and the current build was never installed.  I'd really prefer to just remove any old build that's laying around to avoid problems like this.